bin/helper_upgrade_analyzer

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
Maybe we should call this file `start_upgrade_analyzer`?  "Helper" makes me think of other concepts...